src/lms-harmonizer/tests_integration_pgsql/pgsql_orchestrator.py

```diff
@@ -42,31 +44,27 @@ def _run(config: PgsqlServerConfig, command: List[str]):
 
 
 def run_harmonizer(config: PgsqlServerConfig):
-    _run(
-        config,
-        [
-            "poetry",
-            "run",
-            "python",
-            "edfi_lms_harmonizer",
-            "--server",
-            config.server,
-            "--port",
-            config.port,
-            "--dbname",
-            config.db_name,
-            "--username",
-            config.username,
-            "--password",
-            config.password,
-            "--engine",
-            "postgresql",
-        ],
-    )
```

Comment:
I was getting an error from popen about command not found, seemed to be poetry. Strange that it is working fine in mssql. There's really no reason to run this from poetry anyway, so converted to direct execution of the code.